phpstan.local.neon

```diff
@@ -17568,10 +17568,6 @@ parameters:
     identifier: return.missing
     count: 1
     path: src/Core/Header.php
-  - message: '#^Variable \$vars in isset\(\) always exists and is not nullable\.$#'
-    identifier: isset.variable
-    count: 1
-    path: src/Cqm/QrdaControllers/QrdaReportController.php
```

Comment:
oh oh I edited the neons I submitted yesterday wrong. I though path came first! I'll need to fix. Can I send to you to put back in?

Reply:
Yea, I can add it in.

Reply:
I sent in email both files. No wonder I couldn't fix.

Reply:
Hope I got all the ones in you had done.  I had a few changes I had made in this PR so had to merge the two together.